hc/accounts/tests/test_profile.py

```diff
@@ -18,8 +18,10 @@ def test_it_sends_set_password_link(self):
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
         ### Assert that the token is set
-
+        self.assertEqual(self.alice.profile.token, token)
         ### Assert that the email was sent and check email content
+        self.assertEqual(len(mail.outbox),1)
+        assert "link" in mail.outbox[0].body
```

Comment:
Oh, and there is the issue of running python with the -O flag in an optimized state which loses assert stmts.